sorbet/tapioca/config.yml

```diff
@@ -0,0 +1,13 @@
+gem:
+  # Add your `gem` command parameters here:
+  #
+  # exclude:
+  # - gem_name
+  # doc: true
+  # workers: 5
+dsl:
+  # Add your `dsl` command parameters here:
+  #
+  # exclude:
+  # - SomeGeneratorName
+  # workers: 5
```

Comment:
should we put something here besides the autogenerated thing?

Reply:
There's no further config necessary so I just left it to indicate "untouched". I can delete the comment lines but then we'll just have
```
gem:
dsl:
```
which looks weirder to me 😄

Reply:
Yea, I don't think it hurts to leave the template comments.